src/script/interpreter.h

```diff
@@ -109,11 +110,12 @@ struct PrecomputedTransactionData
 {
     uint256 hashPrevouts, hashSequence, hashOutputs;
     bool m_ready = false;
+    std::vector<CTxOut> m_spent_outputs;
 
     PrecomputedTransactionData() = default;
 
     template <class T>
-    void Init(const T& tx);
+    void Init(const T& tx, std::vector<CTxOut>&& spent_outputs);
```

Comment:
19953: missing changes in `src/validation.cpp`. was that intentional?

Reply:
Yes, attempting to implement them caused crashes and corruptions and the partial was mostly done to resolve conflicts so only a bare-minimum version was implemented, hence the omissions.